specification/synapse/data-plane/Microsoft.Synapse/preview/2021-06-01-preview/runNotebook.json

```diff
@@ -0,0 +1,359 @@
+          "readOnly": true,
+          "type": "string",
+          "description": "Error name."
+        },
+        "evalue": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Error message."
+        },
+        "traceback": {
+          "description": "Error trace.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Result": {
+      "description": "Result of run notebook.",
+      "type": "object",
+      "properties": {
+        "runId": {
+          "type": "string",
+          "description": "Run id."
+        },
+        "runStatus": {
+          "description": "Status of the run notebook.",
+          "type": "string"
+        },
+        "lastCheckedOn": {
+          "description": "Timestamp of last update.",
+          "type": "string"
+        },
+        "sessionId": {
+          "description": "Livy session id.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "sparkPool": {
+          "description": "SparkPool name.",
+          "type": "string"
+        },
+        "snapshot": {
+          "description": "Notebook output.",
+          "type": "object"
+        },
+        "sessionDetail": {
+          "description": "Run notebook session details.",
+          "type": "object"
+        },
+        "exitValue": {
+          "description": "Output of exit command.",
+          "type": "string"
+        },
+        "errors": {
+          "$ref": "#/definitions/Error"
+        }
+      }
+    },
+    "RunNotebookRequest": {
+      "description": "Run notebook request.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Run id (GUID)."
+        },
+        "notebook": {
+          "type": "string",
+          "description": "Notebook name."
+        },
+        "sparkPool": {
+          "type": "string",
+          "description": "SparkPool name."
+        },
+        "sessionOptions": {
+          "description": "Session properties.",
+          "$ref": "./entityTypes/Notebook.json#/definitions/NotebookSessionProperties"
+        },
+        "parameters": {
+          "type": "object",
+          "description": "Run notebook parameters",
+          "additionalProperties": {
+            "$ref": "#/definitions/parameter"
+          }
+        }
+      },
+      "required": [
+        "id",
+        "notebook"
+      ]
+    },
+    "parameter": {
+      "description": "Run notebook parameter.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Parameter type."
+        },
+        "value": {
+          "type": "object",
+          "description": "Parameter value."
+        }
+      }
+    },
+    "RunNotebookResponse": {
+      "description": "Run notebook response.",
+      "type": "object",
+      "properties": {
+        "code": {
```

Comment:
I would not like to expose `code` in SDK. It's redundant with actual HTTP status code.